src/index.css

```diff
@@ -29,26 +29,6 @@ h1 {
   font-style: normal;
 }
 
-button {
-  background-color: #363637;
-  color: #fff;
-  border: none;
-  border-radius: 4px;
-  padding: 8px 16px;
-  cursor: pointer;
-  transition: background-color 0.3s ease;
-  margin: 11px;
-}
-
-button:hover {
-  border-color: #505156;
-}
-
-button:focus,
-button:focus-visible {
-  outline: 4px auto -webkit-focus-ring-color;
-}
```

Comment:
99% sure this was dead css code. You have another css related to button at `Login.css`

```css
button {
  background: linear-gradient(0deg, rgba(2, 0, 36, 1) 0%, rgb(213 76 184) 93%, rgb(145 6 179) 100%);
  color: #fff;
  border: none;
  border-radius: 4px;
  padding: 8px 16px;
  cursor: pointer;
  transition: background-color 0.3s ease;
  margin: 11px;
}

button:hover {
  border-color: #11237f;
  background: #270223;
}

```

so having these lines might led to a css issue (those were using the same selector)